contracts/utils/Panic.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+/**
+ * @dev Helper library for emitting standardized panic codes.
+ */
+// slither-disable-next-line unused-state
+library Panic {
```

Comment:
Given Solidity lists the `0x0` as "Generic", would it make sense to include an?:

```solidity
uint256 internal constant GENERIC = 0x0;
```

Reply:
If we follow solidity list (with naming, then yes)